app/assets/stylesheets/govuk_publishing_components/components/_metadata.scss

```diff
@@ -0,0 +1,70 @@
+.gem-c-metadata {
+  @include core-16;
+  @include responsive-bottom-margin;
+  @extend %contain-floats;
+
+  &.direction-rtl {
+    direction: rtl;
+    text-align: start;
+  }
+
+  dt {
```

Comment:
We should be using a class name here, not a tag. Plenty of other examples of this throughout this file as well.